src/features/singleRight/delegate/ChooseRightsPage/ChooseRightsPage.tsx

```diff
@@ -1,17 +1,266 @@
+/* eslint-disable @typescript-eslint/restrict-template-expressions */
 import * as React from 'react';
 import { useTranslation } from 'react-i18next';
 import { PersonIcon } from '@navikt/aksel-icons';
+import {
+  Alert,
+  Button,
+  Chip,
+  Heading,
+  Ingress,
+  Paragraph,
+  Popover,
+} from '@digdir/design-system-react';
+import { useNavigate } from 'react-router-dom';
+import { useEffect, useState } from 'react';
 
-import { Page, PageContainer, PageContent, PageHeader } from '@/components';
+import { DualElementsContainer, Page, PageContainer, PageContent, PageHeader } from '@/components';
+import { useAppDispatch, useAppSelector } from '@/rtk/app/hooks';
+import { SingleRightPath } from '@/routes/paths';
+import { useMediaQuery } from '@/resources/hooks';
+import {
+  type ChosenService,
+  removeServiceResource,
+} from '@/rtk/features/singleRights/singleRightsSlice';
+import { getSingleRightsErrorCodeTextKey } from '@/resources/utils/errorCodeUtils';
+
+import { RightsActionBar } from './RightsActionBar/RightsActionBar';
+import classes from './ChooseRightsPage.module.css';
+
+interface DelegationResourceDTO {
+  title: string | undefined;
+  serviceIdentifier: string | undefined;
+  action: string;
+}
 
 export const ChooseRightsPage = () => {
   const { t } = useTranslation();
+  const navigate = useNavigate();
+  const dispatch = useAppDispatch();
+  const [selectedRights, setSelectedRights] = useState<DelegationResourceDTO[]>([]);
+  const chosenServices = useAppSelector((state) => state.singleRightsSlice.chosenServiceList);
+  const delegableChosenServices = chosenServices.filter((s) => s.status !== 'NotDelegable');
+  const isSm = useMediaQuery('(max-width: 768px)');
+  let hasPartiallyDelegableAppeared = false;
```

Comment:
Good catch

Reply:
No, when using useState isFirstListElement will automatically change and then the intended functionality will not work since it will always evaluate to false

Reply:
Removed whole thing